test/controllers/books_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class BooksControllerTest < ActionDispatch::IntegrationTest
+  include FactoryGirl::Syntax::Methods
+
+  test 'should get index' do
+    get books_url
+    assert_response :success
+  end
```

Comment:
Ya, the book controller tests are pretty light. I'll see what could be added to make them more thorough.